addon/helpers/optional.js

```diff
@@ -1,8 +1,8 @@
 import { helper } from 'ember-helper';
 
-export function optional([action]) {
+export function optional([action, ...args]) {
   if (typeof action === 'function') {
-    return action;
+    return action.bind(this, ...args);
```

Comment:
Makes sense. Should that be `null` instead, or do we have access here to the calling context?